pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -490,12 +490,19 @@ func (k *Kubernetes) ConfigEnvs(name string, service kobject.ServiceConfig) []ap
 func (k *Kubernetes) CreateKubernetesObjects(name string, service kobject.ServiceConfig, opt kobject.ConvertOptions) []runtime.Object {
 	var objects []runtime.Object
 	var replica int
+
 	if opt.IsReplicaSetFlag || service.Replicas == 0 {
 		replica = opt.Replicas
 	} else {
 		replica = service.Replicas
 	}
 
+	// Check to see if Docker Compose v3 Deploy.Mode has been set to "global"
+	if service.DeployMode == "global" {
+		log.Infof("Global mode has been set. Containers will only be replicated once throughout the cluster")
```

Comment:
This message is still weird. It doesn't say what it does.
For kubernetes Kompose  is ignoring global mode, Kompose just creates regular Deployment with one replica.

So this is not what global mode should do. Should this message just say that global mode is not supported and that regular deployment will be created?